webapp/packages/plugin-data-viewer/src/DataViewerSettingsService.ts

```diff
@@ -73,6 +78,7 @@ export class DataViewerSettingsService extends Dependency {
       createSettingsAliasResolver(this.settingsResolverService, this.settings, {
         'plugin.data-viewer.disableEdit': 'core.app.dataViewer.disableEdit',
         'plugin.data-viewer.disableCopyData': 'core.app.dataViewer.disableCopyData',
+        'plugin.data-viewer.disableDownload': 'core.app.dataViewer.disableDownload',
```

Comment:
Please remove, aliases needed only for old settings, and it's not relevant for the new one